src/components/Map.js

```diff
@@ -27,6 +26,8 @@ const Map = ({
     ).then((res) => setNearbyPlaces(res));
   }, [mapProperties.center.lat, mapProperties.center.lng, setNearbyPlaces]);
 
+  // converts value in seconds to be milliseconds for the setTimeout
+  const fetchDelay = localStorage.getItem("sliderValueInLocalStorage") * 1000;
```

Comment:
Way to remember to convert the units for the delay! Unlike someone who is currently reviewing your PR… 😅

Reply:
I wish I could say that I remembered this on the 1st try. 😬